src/cgame/cg_rocket_draw.cpp

```diff
@@ -75,6 +75,65 @@ void Rocket_UpdateLanguage()
 		hudContext->Update();
 	updateLanguage = false;
 }
+
+// Standard HTML
+
+class LiElement : public Rml::Element
```

Comment:
I guess this is only intended to work for *unordered* lists?

Reply:
Yes, I may rename accordingly, maybe `LuLiElement`?

Reply:
This only works for unordered list but checking if it's in an `ul` or `ol` would be overcomplicated for now, we better want to make all list unordered for now, treating `ol` as `ul` as a fallback, and write complex code the day we support `ol`.